docs/core/install/windows.md

```diff
@@ -82,6 +82,39 @@ The SDK is used to build and publish .NET apps and libraries. Installing the SDK
 > [!div class="button"]
 > [Download .NET SDK](https://dotnet.microsoft.com/download/dotnet)
 
+## ARM-based Windows PCs
+
+The following sections describe things you should consider when installing .NET on an ARM-based Windows PC.
+
+<!-- This section is mirrored in the macos.md file. Changes here should be applied there -->
+
+### What's supported
+
+The following table describes which versions of .NET are supported on an ARM-based Windows PC:
+
+| .NET Version | Architecture | SDK | Runtime | [Path conflict](#path-conflicts) |
+|--------------|--------------|-----|---------|----------------------------------|
+| 6.0          | ARM64        | Yes | Yes     | No                               |
+| 6.0          | x64          | Yes | Yes     | No                               |
+| 5.0          | ARM64        | No  | Yes     | N/A                              |
```

Comment:
I do see Arm64 SDK for .NET 5 at https://dotnet.microsoft.com/download/dotnet/5.0

Reply:
The download page seems to contradict what was documented on the desig doc:
https://github.com/dotnet/designs/blob/main/accepted/2021/x64-emulation-on-arm64/x64-emulation.md#update

Reply:
@richlander do we need to update the design doc in this case?